src/app/utils/url.ts

```diff
@@ -2,6 +2,18 @@
 // `ftp://` is obsolete https://chromestatus.com/feature/6199005675520000
 const validProtocols = ['http:', 'https:', 'ftp:', 'ipfs:', 'data:', 'mailto:']
 
+const twitterHandlePattern = /^@([a-zA-Z0-9_]{1,15})$/
+
+/**
+ * Does this look like a valid Twitter (X) handle?
+ */
+export const isTwitterHandle = (value: string): boolean => twitterHandlePattern.test(value)
```

Comment:
Add preview links to PR description. I couldn't find an example

Reply:
Currently there is no test data, I always faked it locally, so there is no preview link. I am trying to create some ROFL app with the correct data to show... or in worst case I can push a commit with fake data later.

Reply:
https://explorer.dev.oasis.io/testnet/sapphire/rofl/app/rofl1qpcj2zgme3ss0s39zyx5dl36779z54gxkgrqj36q